python/tvm/contrib/rocm.py

```diff
@@ -0,0 +1,51 @@
+"""Utility for ROCm backend"""
+import subprocess
+from . import util
+from ..api import register_func
+
+def rocm_link(in_file, out_file):
+    """Link relocatable ELF object to shared ELF object using lld
+
+    Parameters
+    ----------
+    in_file : str
+        Input file name (relocatable ELF object file)
+
+    out_file : str
+        Output file name (shared ELF object file)
+    """
+    cmd = ["ld.lld"]
+    cmd += ["-shared"]
+    cmd += [in_file]
+    cmd += ["-o"]
+    cmd += [out_file]
+    args = ' '.join(cmd)
```

Comment:
do not do join, simply pass in cmd to Popen, and remove shell=True

Reply:
If I remove `shell=True`, I am getting python runtime error `OSError: [Errno 2] No such file or directory`

Reply:
if shell=True is removed, we cannot pass in args, we should instead pass in cmd (with each argument as a list item of its own